openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js

```diff
@@ -233,7 +238,29 @@ describe('Glossary page should work properly', () => {
       .should('be.visible')
       .click();
 
-    cy.wait('@createGlossary').then(() => {
+    cy.wait('@createGlossary').then(({ request }) => {
+      expect(request.body).to.have.all.keys(
+        'description',
+        'displayName',
+        'mutuallyExclusive',
+        'name',
+        'owner',
+        'reviewers',
+        'tags'
+      );
+      expect(request.body.displayName).equals(NEW_GLOSSARY.name);
+      expect(request.body.name).equals(NEW_GLOSSARY.name);
+      expect(request.body.description).equals(NEW_GLOSSARY.description);
+      expect(request.body.mutuallyExclusive).equals(true);
+      expect(request.body.owner).to.have.all.keys('id', 'type');
+      expect(request.body.reviewers).has.length(1);
+      expect(request.body.tags).has.length(1);
+      expect(request.body.tags[0]).to.deep.equal({
+        labelType: 'Manual',
+        state: 'Confirmed',
+        tagFQN: 'PersonalData.Personal',
+        source: 'Tag',
+      });
```

Comment:
Can't we directly check for object itself in one go?

Reply:
Reviewer and user ID are generated from backend, and everytime its generates new ID and that data we dont have! Please let me know if you have any suggestion!